pom.xml

```diff
@@ -85,7 +85,7 @@
 
     <properties>
         <!-- Artifact versions -->
-        <uk.bom.version>4.0.0-SNAPSHOT</uk.bom.version>
+        <uk.bom.version>4.0.0-pr-232-SNAPSHOT</uk.bom.version>
```

Comment:
This needs fixing before merge to v4?

Reply:
Yes, I will ask Shaun to review again when I change the version back.